File: src/employeemanager/model/FullTimeEmployee.java
package employeemanager.model;

import employeemanager.exceptions.InvalidDataException;
import java.time.LocalDate;

public class FullTimeEmployee extends Employee{

	private double annualSalary;
	private int vacationDays;
	private String benefits;
	
	public FullTimeEmployee(int employeeId,
            String firstName,
            String lastName,
            LocalDate startDate,
            LocalDate dateOfBirth,
            double annualSalary,
            int vacationDays,
            String benefits) throws InvalidDataException {
		super(employeeId, firstName, lastName, startDate, dateOfBirth);
		this.annualSalary = annualSalary;
		this.vacationDays = vacationDays;
		this.benefits = benefits;
	}
	@Override
    public double getPay() {
        return annualSalary / 12.0;
    }

    @Override
    public void reportToManager() {
        System.out.println("[FullTime] " + getFullName() + " reporting to manager.");
        super.reportToManager();
    }
}

File: Part 3/EmployeeManager/models/FullTimeEmployee.cs
using System;

namespace EmployeeManager.Models
{
    public class FullTimeEmployee : Employee
    {
     public double AnnualSalary { get; set; }
     public int VacationDays { get; set; }
     public string Benefits { get; set; }

     public FullTimeEmployee(int employeeId, string firstName, string lastName, DateTime startDate, DateTime dateOfBirth, double annualSalary, int vacationDays, string benefits) : base(employeeId, firstName, lastName, startDate, dateOfBirth)
        {
            AnnualSalary = annualSalary;
            VacationDays = vacationDays;
            Benefits = benefits;
        }

        public override double GetPay() => AnnualSalary / 12.0;

        public override void ReportToManager()
        {
            Console.WriteLine($"[FullTime] {FullName} reporting to manager.");
            base.ReportToManager();
        }   
    }
}